Demo/HYPFormsTests/HYPFormsCollectionViewDataSourceTests.m

```diff
@@ -82,6 +82,50 @@ - (void)testIndexInForms
     [self.dataSource processTarget:[HYPFormTarget showFieldTargetWithID:@"last_name"]];
 }
 
+- (void)testEnableAndDisableTargets
+{
+    HYPFormField *targetField = [self.manager fieldWithID:@"display_name" includingHiddenFields:YES];
+
+    XCTAssertFalse(targetField.disabled);
+
+    HYPFormTarget *disableTarget = [HYPFormTarget disableFieldTargetWithID:@"display_name"];
+
+    [self.dataSource processTarget:disableTarget];
+
+    [self.manager fieldWithID:@"display_name" includingHiddenFields:YES];
```

Comment:
Correct again, oversight on my part. Was pulling my hair yesterday about one of the tests, might have been something related to that.

Removed in https://github.com/hyperoslo/HYPForms/commit/8bf35857685e7143889fc33c08dc9125d6895a46